error: failed to run custom build command for `gecko-media v0.1.0

```
error: failed to run custom build command for `gecko-media v0.1.0 (https://github.com/servo/gecko-media.git#0299b455)`
process didn't exit successfully: `/home/hugh/opensource/servo/target/release/build/gecko-media-59486aef098dd7b4/build-script-build` (exit code: 101)
--- stdout
running: "cmake" "/home/hugh/opensource/servo/.cargo/git/checkouts/gecko-media-6bedb392a81eb7cf/0299b45/." "-DCMAKE_INSTALL_PREFIX=/home/hugh/opensource/servo/target/release/build/gecko-media-ccbcac47a43a429e/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_BUILD_TYPE=Release"
-- Configuring done
-- Generating done
-- Build files have been written to: /home/hugh/opensource/servo/target/release/build/gecko-media-ccbcac47a43a429e/out/build
running: "cmake" "--build" "." "--target" "install" "--config" "Release" "--"
[  9%] Built target stagefright
[  9%] Building C object CMakeFiles/cubeb.dir/gecko/src/media/libcubeb/src/cubeb_pulse.c.o
CMakeFiles/cubeb.dir/build.make:206: recipe for target 'CMakeFiles/cubeb.dir/gecko/src/media/libcubeb/src/cubeb_pulse.c.o' failed
CMakeFiles/Makefile2:115: recipe for target 'CMakeFiles/cubeb.dir/all' failed
Makefile:127: recipe for target 'all' failed

--- stderr
/home/hugh/opensource/servo/.cargo/git/checkouts/gecko-media-6bedb392a81eb7cf/0299b45/gecko/src/media/libcubeb/src/cubeb_pulse.c:11:30: fatal error: pulse/pulseaudio.h: No such file or directory
compilation terminated.
make[2]: *** [CMakeFiles/cubeb.dir/gecko/src/media/libcubeb/src/cubeb_pulse.c.o] Error 1
make[1]: *** [CMakeFiles/cubeb.dir/all] Error 2
make: *** [all] Error 2
thread 'main' panicked at '
command did not execute successfully, got: exit code: 2

build script failed, must exit now', .cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.27/src/lib.rs:627:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/README.md
+++ b/README.md
@@ -56,7 +56,8 @@ sudo apt install git curl freeglut3-dev autoconf libx11-dev \
     libfreetype6-dev libgl1-mesa-dri libglib2.0-dev xorg-dev \
     gperf g++ build-essential cmake virtualenv python-pip \
     libssl1.0-dev libbz2-dev libosmesa6-dev libxmu6 libxmu-dev \
-    libglu1-mesa-dev libgles2-mesa-dev libegl1-mesa-dev libdbus-1-dev
+    libglu1-mesa-dev libgles2-mesa-dev libegl1-mesa-dev libdbus-1-dev \
+    libpulse-dev
 ```
 
 If you using a version prior to **Ubuntu 17.04** or **Debian Sid**, replace `libssl1.0-dev` with `libssl-dev`.
@@ -72,7 +73,7 @@ sudo dnf install curl freeglut-devel libtool gcc-c++ libXi-devel \
     freetype-devel mesa-libGL-devel mesa-libEGL-devel glib2-devel libX11-devel libXrandr-devel gperf \
     fontconfig-devel cabextract ttmkfdir python python-virtualenv python-pip expat-devel \
     rpm-build openssl-devel cmake bzip2-devel libXcursor-devel libXmu-devel mesa-libOSMesa-devel \
-    dbus-devel ncurses-devel
+    dbus-devel ncurses-devel pulseaudio-libs-devel
 ```
 #### On CentOS
 
@@ -81,7 +82,7 @@ sudo yum install curl freeglut-devel libtool gcc-c++ libXi-devel \
     freetype-devel mesa-libGL-devel mesa-libEGL-devel glib2-devel libX11-devel libXrandr-devel gperf \
     fontconfig-devel cabextract ttmkfdir python python-virtualenv python-pip expat-devel \
     rpm-build openssl-devel cmake bzip2-devel libXcursor-devel libXmu-devel mesa-libOSMesa-devel \
-    dbus-devel ncurses-devel python34
+    dbus-devel ncurses-devel python34 pulseaudio-libs-devel
 ```
 
 #### On openSUSE Linux

